### PR TITLE
[document_repository] Notification emails 

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
Adding the Notification for Uploading a new file and Creating a new category.
#### Testing instructions (if applicable)
Test it on testing VM. Uploading a new file and Creating a new category, use who has these notification service will recive the email.
#### Link(s) to related issue(s)
https://github.com/aces/Loris/issues/6580
* Resolves #  (Reference the issue this fixes, if any.)

### CODE REVIEW

**Comment by kongtiaowang** [comment]
reopen this PR, travis does not restart for this PR.

**Comment by cmadjar** [comment]
@kongtiaowang Travis still has not started for that PR

**Comment by kongtiaowang** [comment]
@cmadjar It pass the Travis right now.

File: modules/document_repository/php/files.class.inc [hunk 1]
```diff
@@ -111,12 +111,24 @@ class Files extends \NDB_Page
...
         try {
             $db->update(
                 'document_repository',
                 $updateValues,
                 ['record_id' => $req['id']]
             );
+            $factory  = \NDB_Factory::singleton();
+            $baseURL  = $factory->settings()->getBaseURL();
+            $msg_data = [
+                'updatedDocument' => $baseURL. "/document_repository/",
```
Comment (driusan) [inline modules/document_repository/php/files.class.inc:128]:
```suggestion
                'updatedDocument' => $baseURL . "/document_repository/",
```

File: modules/document_repository/php/uploadcategory.class.inc [hunk 2]
```diff
@@ -89,5 +89,16 @@ class UploadCategory extends \NDB_Page
...
             ]
         );
+        $Notifier = new \NDB_Notifier(
+            "document_repository",
+            "new_category"
+        );
+        $factory  = \NDB_Factory::singleton();
+        $baseURL  = $factory->settings()->getBaseURL();
+        $msg_data = [
+            'newCategory' => $baseURL. "/document_repository/",
```
Comment (driusan) [inline modules/document_repository/php/uploadcategory.class.inc:99]:
```suggestion
            'newCategory' => $baseURL . "/document_repository/",
```

**Pull request merged**